resources/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -16,6 +16,17 @@ spec:
         app.kubernetes.io/name: {{ include "kyma-env-broker.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      {{ if .Values.global.isLocalEnv }}
+      hostAliases:
```

Comment:
could you add a comment why we need that and why all of those hostnames and not only some of them?